docker/Makefile

```diff
@@ -84,10 +86,11 @@ dkr-no-deps:
 run-test:
 	@docker run --rm \
     -v $(CODE):/code \
-    $(DKR_IMG) pytest --timeout=30 .
+    $(DKR_IMG) with-test-db env AWS_DEFAULT_REGION=us-west-2 \
```

Comment:
@Kirill888 so make me smart, running tests isn't an entirely local process, it touches AWS as well?

Reply:
No we do not connect to aws, we use moto library, but it still needs default region defined, although it should probably be defined in a fixture and not here.

I do think that some tests reach out for external resources via http, so it's not fully self contained, we should fix that

Reply:
Roger, sounds good. I know pystac-client has had good success with https://pytest-vcr.readthedocs.io/en/latest/ for robustifying network-y tests.

Reply:
that one looks cool, we just use `pytest.httpserver`, but not everywhere, some places grab files from github, making tests easy to write is a major undertaking